python/chemiscope/sphinx/nodes.py

```diff
@@ -41,14 +45,19 @@ def depart_chemiscope_html(self, node):
 
 
 def generate_html_content(
-    filepath, mode="default", include_headers=True, static_path="_static"
+    filepath,
+    mode="default",
+    warning_timeout=4000,
```

Comment:
I think we should pick one default value for the timeout and use it everywhere. It looks like `4000` and `10000` are used in different places

Reply:
so the logic here is that in jupyter you might be debugging and so you want messages to stay up longer... maybe overkill but imo is better to tweak persistence to the use case